Formula/gitui.rb

```diff
@@ -16,14 +16,25 @@ class Gitui < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "54b24c5b05ac59a33c54cf7fb310aae65a7dd92d9d707c01d2b6bd310b831cd2"
   end
 
+  depends_on "pkg-config" => :build
   depends_on "rust" => :build
-
+  depends_on "libgit2"
+  depends_on "openssl@1.1"
```

Comment:
I think we may want to delay doing this until we've migrated everything to `openssl@3`.